File: miniMaxSum.go
package main

import (
	"fmt"
)

func MiniMaxSum(arr []int32) {

	for i := 0; i < len(arr); i++ {
		// fmt.Println(arr[i])

		for j := 0; j < len(arr); j++ {
			if arr[i] != arr[j] {
				fmt.Println(arr[i])
			}
		}

	}
}

File: js/miniMaxSum.js
function miniMaxSum(arr) {
  let sum = 0;
  let total = [];
  // Write your code here
  for (let i = 0; i < arr.length; i++) {
    sum += arr[i];
    total.push(sum);
  }

  total.map((value, index) => {
    console.log(arr[index]);
    // value -= arr[index];
    // console.log(value);
  });
}

miniMaxSum([7, 69, 2, 221, 8974]);
